models/klaviyo__flows.sql

```diff
@@ -0,0 +1,37 @@
+with flow as (
+
+    select *
+    from {{ var('flow') }}
+),
+
+flow_metrics as (
+
+    select *
+    from {{ ref('int_klaviyo__campaign_flow_metrics') }}
+    where last_touch_flow_id is not null
+),
+
+flow_join as (
+    
+    {% set exclude_fields = [ 'last_touch_campaign_id', 'last_touch_flow_id'] %}
+    {% set exclude_fields = exclude_fields | upper if target.type == 'snowflake' else exclude_fields %}
+
+    select
+        flow.*,
+        {{ dbt_utils.star(from=ref('int_klaviyo__campaign_flow_metrics'), except=exclude_fields) }}
+
+    from flow
+    left join flow_metrics on flow.flow_id = flow_metrics.last_touch_flow_id
+),
+
+final as (
+
+    select 
+        *,
```

Comment:
FYI the unique_counts for this model are very off. Related to something upstream, but heads up that this is impacted as well.